### DISCUSSION TITLE
Creating a file and calling Gephi

### DISCUSSION THREAD

**Discussion by provalis** [gephi/gephi#2770] [Q&A]
Is there a way to create either a matrix data or an edge data file with an extension associated with Gephi so that double-clicking on this file will automatically call Gephi.  I want to be able to call Gephi automatically from another application. 

I looked at a sample .gephi file (renamed to .zip) but the data seems to be stored in a binary file along with other XML files.

NP

**Comment by mbastian** [comment]
Hi @provalis, if I understand correctly you would like your application to create .gephi files? That is currently not possible as that format is not open and documented. But the [GEXF format](https://gexf.net/) has a lot of features and maybe can serve your needs. In general, on Windows you can associate files formats with Gephi easily. The installer proposes you this, so that Gephi is opened when dbl-clicking on these formats. On Mac OS it doesn't work (see https://github.com/gephi/gephi/issues/2594) thought.

**Reply by provalis** [reply]
Thanks, this is exactly the solution I wanted.  I assume GEXF can be associated with other applications as well.  We will explore this and implement the GEXF file format.

NP
